cni/pkg/cmd/root.go

```diff
@@ -246,8 +246,10 @@ func constructConfig() (*config.Config, error) {
 		CNIConfName:      viper.GetString(constants.CNIConfName),
 		ChainedCNIPlugin: viper.GetBool(constants.ChainedCNIPlugin),
 
-		// make plugin (which runs out-of-process) inherit our current log level
-		PluginLogLevel:        istiolog.LevelToString(istiolog.FindScope(constants.CNIAgentLogScope).GetOutputLevel()),
+		// Whatever user has set (with --log_output_level) for 'cni-plugin', pass it down to the plugin. It will use this to determine
```

Comment:
(largely reiterating what I've said before for context):

I am more inclined to simply drop the scoping entirely to keep the plugin simple, and let the log receiver decide what to log based on its level. 

We don't intend for plugin logs to go to a different logsink than the agent logs, ultimately, so having a separate scope is largely moot - if the issue is that the plugin logs are spammy/useless and we want to hide them, better to fix that than use scopes to mask it.

Especially since no one will ever use this sub-component scoping for any other purpose, and it would confuse people more if we documented it versus simply dropped it and stuck with a single scope for the whole of `istio-cni`.

But tomato tomahto, and it's already here, so it should work at least. TY for fixing.

Reply:
I want spammy logs from the plugin but not the agent today, and tomorrow I want spammy logs from the agent and not the plugin. There is no need to force people into all or nothing :slightly_smiling_face: